Makefile

```diff
@@ -0,0 +1,49 @@
+artifact_name       := overseas-entities
```

Comment:
i think artifact name should be overseas-entities-web

Reply:
See: https://github.com/companieshouse/confirmation-statement-web/pull/6/files

Reply:
ah, looks like it was corrected since then - https://github.com/companieshouse/confirmation-statement-web/blob/main/Makefile